src/http/request_handler.rs

```diff
@@ -113,6 +114,26 @@ pub async fn handle_request<T: DeserializeOwned + GraphQLRequestLike, Http: Http
   state: Arc<AppContext<Http, Env>>,
 ) -> Result<Response<Body>> {
   match *req.method() {
+    hyper::Method::POST
+      if state.blueprint.server.enable_showcase && req.uri().path().ends_with("/showcase/graphql") =>
+    {
+      let server_ctx = match showcase_get_app_ctx::<T, _, DummyFileIO, _>(
+        &req,
+        (
+          state.universal_http_client.clone(),
+          DummyEnvIO,
+          None,
+          state.cache.clone(),
+        ),
+      )
+      .await?
+      {
+        Ok(server_ctx) => server_ctx,
+        Err(res) => return Ok(res),
+      };
+
+      graphql_request::<T, _, _>(req, &server_ctx).await
+    }
     hyper::Method::POST if req.uri().path().ends_with("/graphql") => {
```

Comment:
This `POST /graphql` is in our hot path and should be the first route we should check.

Reply:
The endpoint is not `POST /graphql`, it's `POST */graphql`, since we're using `ends_with`. Checking it first would require us to check that the endpoint ends with `/graphql`, **and that the endpoint doesn't end with `/showcase/graphql`**, otherwise we can't check for the showcase. This comparison has to be run anyways, so changing the order of the match wouldn't make any difference.

Reply:
We could change the showcase endpoint to just be `/showcase` so that it doesn't collide with the `/graphql` check